hydra-node/src/Hydra/Ledger/Simple.hs

```diff
@@ -27,6 +35,12 @@ data SimpleTx = SimpleTx
   }
   deriving stock (Eq, Ord, Generic, Read, Show)
 
+instance ToJSON SimpleTx where
+  toJSON = genericToJSON (aesonPrefix camelCase)
```

Comment:
So this is creating a JSON object with fields `id`, `inputs` and `outputs`? IMO this "drop all lowercase letters until first uppercase" convention of this `aesonPrefix` is only sligthly better than `drop 2` and would like to avoid such things as much as possible. Can we stick to just use field names as is and make then non-overlapping or non-conflicting on a case by case basis?

Reply:
> Can we stick to just use field names as is and make then non-overlapping or non-conflicting on a case by case basis?

I am fine with that ^.^